main.js

```diff
@@ -3893,6 +3908,24 @@ function init(compactGroupId) {
         logger.info(hostLogPrefix + ' ' + tools.appName + '.js-controller version ' + version + ' ' + ioPackage.common.name + ' starting');
     }
 
+    const pluginSettings = {
+        scope: 'controller',
+        namespace: hostObjectPrefix,
+        logNamespace: hostLogPrefix,
+        log: logger,
+        iobrokerConfig: config,
+        parentPackage: null
+    };
+    try {
+        pluginSettings.parentPackage = JSON.parse(fs.readFileSync(__dirname + '/package.json', 'utf8'));
+    }
+    catch (err) {
+        logger.error(hostLogPrefix + ' Can not read js-controller package.json');
```

Comment:
Is the plugin handler supposed to work if the js-controller package.json cannot be read? Which info does it need from there?

Reply:
The package content is passed to the plugin. Sentry uses data from there (mostly name and version fields) but yes should work without

Reply:
So the plugins would all need to check if the package was actually passed? I would prefer it if we did not load the plugins in this edge case (package.json missing or broken).

Reply:
Hm ... it is initialized with an empty object if not there ....